component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryActions.java

```diff
@@ -1,167 +1,44 @@
-/*
```

Comment:
This class was renamed.  The code that is removed here is the old code.  The old accessor is named AuditEntryActionsLegacy.